multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```diff
@@ -29,10 +34,32 @@ spec:
       labels:
         app: {{.Values.gateway.name}}
     spec:
+      {{- if .Values.enablePodAntiAffinity -}}
+      {{- $local := dict "component" "gateway" -}}
+      {{- include "linkerd.pod-affinity" $local | nindent 6 -}}
+      {{- end }}
       containers:
         - name: pause
           image: gcr.io/google_containers/pause
       serviceAccountName: {{.Values.gateway.name}}
+{{- if .Values.enablePodAntiAffinity }}
+---
+kind: PodDisruptionBudget
+apiVersion: policy/v1beta1
+metadata:
+  name: linkerd-dst
+  namespace: {{.Values.namespace}}
+  labels:
+    linkerd.io/control-plane-component: gateway
+    linkerd.io/control-plane-ns: {{.Values.namespace}}
```

Comment:
The control plane usually lives in a different namespace (`linkerd` vs `linkerd-multicluster`), but we're not using this label anyways, so better to get rid of it.

Reply:
Good point. Removed 👍